### ISSUE TITLE
Visual C++ runtime update (v0.12.0)

### ISSUE THREAD

**Issue by dotas86** [gerwaric/acquisition#84]
Hi, it's me again!

I installed latest version 0.12 and I did update Visual C++ Runtime. Now when I open it I have the following error: 
The Microsoft Visual C++ Runtime needs to be updated.
Please re-install acquisition with this option enabled.

I did, twice and nothing. I should also mention that Avast did try to see if the file was a virus and sent it for a checkup. Maybe that messed it.

I  tried the beta and that one works for me.

**Comment by gerwaric** [comment]
In case you're interested, the problem is that Microsoft made a change to their DLLs in 2022 that has been causing crashes if a user does not update their DLLs. However, Microsoft doesn't do any checks for this, so I tried to add some myself, which aren't working for you.

**Comment by dotas86** [comment]
Classic Microsoft for you!

I forgot to mention that it also said that the vc_redistx64 could not be found in the directory of acquisition. Maybe that's why it did not work.

**Comment by gerwaric** [comment]
@dotas86 I've manually updated the installer so that it includes the vc_redist executable. Can you download the v0.12.0 again at let me know if the installer works with the MSVC runtime option set?

**Comment by dotas86** [comment]
It  does work. Some other things that came up after:

After the installation of the MSVC the PC force restarted. I didn't mind since I wasn't doing anything, but someone might have some work related things open that may be influenced by the force restart.

Second thing is that using OAuth I couldn't update the tabs, I had to login with POESESSID in order to work.

I know the second is not related to the previous problem, but since I found it I thought to write it here.
